document_page_quality_manual/README.rst

```diff
@@ -1,3 +1,6 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
@elicoidal Do you mean anything else in addition to the target of the badge link?